### PR TITLE
ART-7776 Skip nag-upstream for groups not in 'release' state

### PR DESCRIPTION
**Pull request by locriandev**
Context: https://redhat-internal.slack.com/archives/GDBRP5YJH/p1694594567569429?thread_ts=1694591076.556999&cid=GDBRP5YJH

### CODE REVIEW

File: jobs/build/nag-upstream/Jenkinsfile [hunk 1]
```diff
@@ -34,39 +34,47 @@ node() {
...
-                report = buildlib.doozer("${doozerOpts} images:streams prs list", [capture: true]).trim()
-                if (report) {
-                    data = readYaml text: report
-                    text = ""
-                    data.each { email, repos ->
-                        text += "*${email} is a contact for these PRs:*\n"
-                        repos.each { _, prs ->
-                            prs.each { pr -> text += ":black_small_square:${pr.pr_url}\n" }
+    // Skip the build for versions in eol/pre-release state
+    def out = buildlib.doozer("--group=openshift-${build_version} config:read-group --yaml software_lifecycle", [capture: true]).trim()
```
Comment (thegreyd) [inline jobs/build/nag-upstream/Jenkinsfile:38]: build_version -> BUILD_VERSION ?

Comment (thegreyd) [reply jobs/build/nag-upstream/Jenkinsfile:38]: and we can use doozerOpts here?

Comment (locriandev) [reply jobs/build/nag-upstream/Jenkinsfile]: Fixed

**Pull request merged**